tests/visual/code.html

```
@@ -73,26 +74,24 @@ <h2>行高亮和聚焦</h2>
+    <title>Quick Start - Layui</title>
+    <link href="//unpkg.com/layui/dist/css/layui.css" rel="stylesheet">
+  </head>
+  <body>
+    <!-- HTML Content -->
+    <script src="//unpkg.com/layui/dist/layui.js"></script>
+    &lt;script type="module"&gt;
+      const { layer } = layui;
+
+      // Welcome
+      layer.msg('Hello World', { icon: 6 });
+    </script>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </script> ]

(tag-pair)